packages/server/src/server.ts

```diff
@@ -133,7 +133,7 @@ export const createApp = async (options: CreateAppOptions): Promise<{
 		});
 	});
 
-	const server = httpolyglot.createServer(options.bypassAuth ? {} : options.httpsOptions || certs, app) as http.Server;
+	const server = httpolyglot.createServer(options.allowHttp ? {} : options.httpsOptions || certs, app) as http.Server;
```

Comment:
This change causes `(SSL: error:14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure:SSL alert number 40)` when running code-server behind a reverse proxy which provides SSL. Although this is likely due to faulty configuration of the reverse proxy. In nginx, I had to change
```
location / {
       proxy_pass https://localhost:8443;
       [...]
```
to
```
location / {
       proxy_pass http://localhost:8443;
       [...]
```

Just posting this here in case anyone mis-configured their reverse proxy like I did ;)